xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -346,10 +440,18 @@
         <Parameter Name="unmanagedDllName" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="unmanagedDllName">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="unmanagedDllName">Name of the native unmanaged DLL to be loaded without its extension</param>
```

Comment:
"native" and "unmanaged" are effectively synonims in this context. I would just use "unmanaged" to follow the API naming scheme.
There's no rule how the name should look like - extension or not. In fact this can be a completley random string...
Typically it is the name without extension, but not always.